packages/@webex/plugin-meetings/src/meeting/brbState.ts

```diff
@@ -95,6 +95,13 @@ export class BrbState {
     return this.sendLocalBrbStateToServer(sendSlotManager)
       .then(() => {
         this.state.syncToServerInProgress = false;
+
+        // This is a workaround for the fact that the server does not send the brb state
+        // in the locus update when a user joins from multiple devices but not all devices are requested brb.
+        // In the future, this could be improved with a new brb locus update handler
+        // https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-655626
+        this.handleServerBrbUpdate(this.state.client.enabled);
```

Comment:
as discussed offline, it feels like we should really understand better what is the desired behavior in these cases, for example even if we force our brb status to "on" locally here, if Locus doesn't think it's on, then what will the other meeting participants see as our status? And probably worth discussing this with Colin to confirm what is the desired behaviour for VDI.

Also, it would be good to update some unit test to add the check that this call is being made.